tests/test_postgres_fields.py

```diff
@@ -493,8 +493,6 @@ def test_value_DateTimeRangeTZ_invalid_non_json_parsable(self):
 			test_model = TimeTable(daterange=DateTimeTZRange(lower, upper))
 			test_model.save()
 
-		self.assertEqual("expected string or bytes-like object", str(te.exception))
```

Comment:
Do these test still make sense if we remove this assert? Otherwise I suggest we either update or remove them.

Reply:
Yes, a few lines above we're catching the exception and validating its type. Only the exception message checks were removed because they're inconsistent across methods and Python versions.